djangocms_page_meta/cms_toolbars.py

```diff
@@ -44,7 +48,11 @@ def populate(self):
         permission = self.request.current_page.has_change_permission(self.request.user)
         can_change = self.request.current_page and permission
         if has_global_current_page_change_permission or can_change:
-            not_edit_mode = not self.toolbar.edit_mode
+            try:
+                not_edit_mode = not self.toolbar.edit_mode
+            except AttributeError:
+                not_edit_mode = not self.toolbar.edit_mode_active
```

Comment:
I'd reverse the order of these, to keep the old version as a fallback. It makes easier to remove it once we drop django CMS 3.4

Reply:
thanks :+1: